.env.example

```
@@ -1,36 +1,4 @@
-
-# Deployed Contract Addresses (filled after deployment)
-ZKPOR_VERIFIER=deployed_zkPoRVerifier_contract_address
-BLEU_MINT_CONTROLLER=deployed_BleuCrownMintUltraMax_contract_address
-CONTRACT=legacy_contract_address_for_compatibility
-
-# Minting Configuration
-RECIPIENT=address_to_receive_minted_nfts
-CID=ipfs_content_id_for_metadata
+INFURA_API_KEY=your_infura_project_id
+PRIVATE_KEY=your_private_key
+MUMBAI_RPC_URL=https://matic-mumbai.chainstacklabs.com
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The MUMBAI_RPC_URL key should go before the PRIVATE_KEY key

(UnorderedKey)